gofsutil_mount_linux.go

```diff
@@ -344,59 +376,87 @@ func (fs *FS) getMountInfoFromDevice(
 	}
 
 	var cmd string
+	var output string
 	lsblkNew, err := fs.isLsblkNew()
 	if err != nil {
 		return nil, err
 	}
+	log.Infof("device ID from getMountInfoFromDevice is %s", devID)
 	/* #nosec G204 */
-	checkCmd := "lsblk -P | awk '/mpath.+" + devID + "/ {print $0}'"
```

Comment:
Should this choose either  mpath or ppath devices based on the config.

Reply:
yes Nida , based on the device type we search the device name .